File: boots/src/main/java/com/project/boots/models/Product.java
package com.project.boots.models;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "products")
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "product_id")
    private int productId;

    @Column(name = "image_url")
    private String imageUrl;

    @Column(name = "title")
    private String title;

    @Column(name = "info")
    private String info;

    @Column(name = "sizes")
    private String sizes;

    @Column(name = "price")
    private int price;

}

File: practiceProj/src/app/classes/product.ts
export class Product {

    constructor(
        public productId: number,
        public imageUrl: string,
        public title: string,
        public sizes: string,
        public price: number,
        public info?: string
    ) {}

    // productId: number;
    // imageUrl: string;
    // title: string;
    // info: string;
    // sizes: string;
    // price: number;
}
